test/juttled/juttled.spec.js

```diff
@@ -173,7 +173,7 @@ describe("Juttled Tests", function() {
                     return fs_extra.copyAsync(__dirname + "/forever.juttle", not_readable);
                 })
                 .then(function() {
-                    return fs.chmodAsync(not_readable, 0300);
+                    return fs.chmodAsync(not_readable, 0o300);
```

Comment:
What is that? I trust you, just don't know what it is, or how to even google for it :-).